src/client/components/app/Footer/index.tsx

```diff
@@ -107,93 +81,64 @@ const StyledIconLink = styled(Link)`
 `;
 
 const StyledIcon = styled(Icon)`
-  width: var(--icon-size);
-  height: var(--icon-size);
-  fill: ${({ theme }) => theme.colors.secondary};
-`;
-
-const LogoSection = styled.a`
-  display: flex;
-  justify-content: flex-end;
-`;
-
-const AlchemyLogo = styled(AlchemyCertified)`
-  width: 17.2rem;
-  fill: ${({ theme }) => theme.colors.secondary};
+  width: ${SocialSectionIconSize};
+  height: ${SocialSectionIconSize};
+  fill: ${({ theme }) => theme.colors.icons.variant};
 `;
 
 const StyledFooter = styled.footer`
   display: grid;
-  grid-template-columns: 1fr 2fr 1fr;
+  grid-template-columns: auto auto;
   align-items: center;
   width: 100%;
   flex-wrap: wrap;
   gap: 1rem;
   row-gap: 1.5rem;
-  padding: 4rem;
+  padding: ${({ theme }) => theme.card.padding};
   border-radius: ${({ theme }) => theme.globalRadius};
-  background-color: ${({ theme }) => theme.colors.surface};
   max-width: ${({ theme }) => theme.globalMaxWidth};
-  margin-top: ${({ theme }) => theme.layoutPadding};
 
   @media ${device.tablet} {
     grid-template-columns: 1fr;
     padding: 2rem;
 
     ${SocialSection},
-    ${LinkSection},
-    ${LogoSection} {
+    ${LinkSection} {
       justify-content: center;
     }
   }
 
   ${StyledLink},
-  ${StyledIconLink},
-  ${AlchemyLogo} {
+  ${StyledIconLink} {
     transition: filter 200ms ease-in-out;
-    filter: opacity(50%);
+    filter: opacity(100%);
 
     &:hover {
-      filter: opacity(100%);
+      filter: opacity(70%);
     }
   }
 `;
 
-const socialIcons = (
-  <SocialSection>
-    {socialLinks.map((social, index) => {
-      return (
-        <StyledIconLink href={social.link} target="_blank" key={index}>
-          <StyledIcon Component={social.icon} />
-        </StyledIconLink>
-      );
-    })}
-  </SocialSection>
-);
-
 export const Footer = ({ className }: FooterProps) => {
   const { t } = useAppTranslation('common');
 
   return (
     <StyledFooter className={className}>
-      {socialIcons}
-
       <LinkSection>
-        {footerLinks.map((footerLink) => {
-          return (
-            <StyledLink href={footerLink.link} key={footerLink.name} target="_blank">
-              {t(`footer.links.${footerLink.name}`)}
-            </StyledLink>
-          );
-        })}
+        {footerLinks.map((footerLink) => (
+          <StyledLink href={footerLink.link} key={footerLink.name} target="_blank">
+            {t(`footer.links.${footerLink.name}`)}
+          </StyledLink>
+        ))}
       </LinkSection>
 
-      <LogoSection
-        href="https://dashboard.alchemyapi.io/signup?referral=c642981b-19e0-45e9-a169-0b80b633992b"
-        target="_blank"
-      >
-        <AlchemyLogo />
-      </LogoSection>
+      <SocialSection>
+        {socialLinks.map((social, index) => (
+          <StyledIconLink href={social.link} target="_blank" key={index}>
```

Comment:
We should avoid using the index for keys https://reactjs.org/docs/lists-and-keys.html#keys

I think the link is a stable id for this
```suggestion
          <StyledIconLink href={social.link} target="_blank" key={social.link}>
```

Reply:
I don't think it is a stable id, since it is a normal string that can be repeated pretty easily without any checks. maybe social.link-index would do if you don't like to let only the index

Reply:
All good, let's leave as is